napari/utils/key_bindings.py

```diff
@@ -316,6 +317,29 @@ def __init_subclass__(cls, **kwargs):
 
     bind_key = KeybindingDescriptor(bind_key)
 
+    @classmethod
+    def register_action(cls, *, description=None, name=None):
+        """
+        Convenient decorator to register an action with the current KeymapProvider
+
+        It will use the function name as the action name, and the function docstring as
+        the function description.
+        """
+        from .action_manager import action_manager
+
+        def _inner(func):
+            nonlocal name
+            nonlocal description
+            if name is None:
+                name = func.__name__
```

Comment:
As above: should this perhaps be `'.'.join([func.__module__, func.__qualname__])`?

Reply:
This has been remove in the other PR to have less magic, and requires explicit names.